cihp13/cihp13_metadata_BARRA2.yml

```
@@ -0,0 +1,91 @@
+# Apply file attributes defined for the QDC scaled CMIP6 data which feeds into CIHP-13.
+#
+# This is for BARRA2 input.
+# metadata that would be nice to add but don't fit this YAML format:
+#  - global:
+#    - date_created
+
+global_keep:
+  - geospatial_lat_min
+  - geospatial_lat_max
+  - geospatial_lon_min 
+  - geospatial_lon_max
+
+global_overwrite:
+  - title: 'QDC-Scaled CMIP6 Application-Ready Climate Projections'
+  - Conventions: 'CF-1.8, ACDD-1.3'
+  - summary: 'Application-ready climate projections are available for daily maximum and minimum temperature, precipitation, solar radiation, relative humidity and wind speed. The data have been created by applying future climate changes simulated by a selection of CMIP6 Global Climate Models to BARRA2 reanalysis data using the QDC scaling method.'

```

@malcolmking Maybe we shouldn't l
```suggestion
  - summary: 'Application-ready climate projections. The data have been created by applying future climate changes simulated by a selection of CMIP6 Global Climate Models to BARRA2 reanalysis data using the QDC scaling method.'

```

---

@malcolmking I'm thinking maybe we shouldn't list all the variables in case we add/remove variables from the dataset at some point?

---

Good idea, I agree that that's probably a good idea (thanks for updating the correct short version for BARRA version 2 as well).